pallets/ajuna-board/src/lib.rs

```diff
@@ -74,6 +77,14 @@ pub mod pallet {
 		/// Maximum number of players
 		#[pallet::constant]
 		type MaxNumberOfPlayers: Get<u32>;
+
+		#[pallet::constant]
+		/// Maximum number of successive blocks with no player activity
+		type MaxNumberOfIdleBlocks: Get<u32>;
```

Comment:
Just a tiny nitpick :smile: 
```suggestion
		/// Maximum number of successive blocks with no player activity
		#[pallet::constant]
		type MaxNumberOfIdleBlocks: Get<u32>;
```